crates/cast/bin/cmd/wallet/mod.rs

```diff
@@ -135,6 +135,10 @@ pub enum WalletSubcommands {
         /// (~/.foundry/keystores)
         #[clap(long, short)]
         keystore_dir: Option<String>,
+        /// Password for the JSON keystore in cleartext
+        /// This is unsafe, we recommend using the default hidden password prompt
+        #[clap(long, env = "CAST_PASSWORD", value_name = "PASSWORD")]
```

Comment:
should this maybe be `CAST_UNSAFE_PASSWORD`?

Reply:
`CAST_PASSWORD` was chosen to have consistency with `cast wallet new`, but I'm more than happy to make it `CAST_UNSAFE_PASSWORD` too.